fhirAssembly/src/main/resources/assembly/com.vantiq.fhir.fhirConnection/procedures/com/vantiq/fhir/fhirService_bundleInteraction.vail

```diff
@@ -0,0 +1,15 @@
+// Process a Bundle interaction (typically bundle type 'batch' or 'transaction' but others may be supported by the server)
+
+package com.vantiq.fhir
+PROCEDURE fhirService.bundleInteraction(bundle Object REQUIRED DESCRIPTION "The bundle to be invoked.",
+                                        modifiers com.vantiq.fhir.Modifiers DESCRIPTION "Query modifiers and headers for the call."): 
+                      com.vantiq.fhir.FHIRResponse WITH updateInterface=true
```

Comment:
We usually reserve `updateInterface=true` for generated use cases.  Have you been using this all along or is this newish (I honestly can't remember).

Reply:
Well, it's not new to this PR I don't think.  Alex suggested it to allow for descriptions, etc., to get updated.  I can remove them in the next PR if that's better -- I don't want to misuse things.

Reply:
It's not a misuse per-se, just odd.  My major concern is that we don't document it and now it is creeping closer to the user.  Does the UI not update the descriptions when you edit the procedure sig?  It probably should, so I would log an issue/Aha for that if it doesn't.  Of course, that assumes you are using the UI to create any of these, I figured you were given the project descriptions, but...